### ISSUE TITLE
Steam Overlay Error

### ISSUE THREAD

**Issue by ashuntu** [canonical/steam-snap#24]
Consistently on every with Integrated Intel graphics, there is the following overlay error (or variants of it):

```
ERROR: ld.so: object '/home/a/snap/steam/common/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
```

On some games overlay notifications are blank or not rendered correctly, but I haven't found a consistent pattern.

`snap info steam`
```
tracking:     latest/edge
refresh-date: 5 days ago, at 14:33 CDT
channels:
  latest/stable:    –                              
  latest/candidate: –                              
  latest/beta:      1.0.0.74 2022-05-17 (17) 269MB -
  latest/edge:      1.0.0.74 2022-05-27 (19) 272MB -
installed:          1.0.0.74            (19) 272MB -
```

**Comment by bdwasd** [comment]
same problem, dota not launching from steam (snap version), but if launch from steam (apt version) it is work

**Comment by aztrock** [comment]
same problem in arch linux

```
                   -`                 
                  .o+`                 aztrock@msi-aztrock-arch
                 `ooo/                 OS: Arch Linux 
                `+oooo:                Kernel: x86_64 Linux 6.0.7-arch1-1
               `+oooooo:               Uptime: 10m
               -+oooooo+:              Packages: 1021
             `/:-:++oooo+:             Shell: zsh 5.9
            `/++++/+++++++:            Resolution: 4080x1920
           `/++++++++++++++:           WM: i3
          `/+++ooooooooooooo/`         GTK Theme: Pop-dark [GTK2/3]
         ./ooosssso++osssssso+`        Icon Theme: Papirus
        .oossssso-````/ossssss+`       Font: MesloLGL Nerd Font Mono 9
       -osssssso.      :ssssssso.      Disk: 261G / 919G (30%)
      :osssssss/        osssso+++.     CPU: AMD Ryzen 5 5600X 6-Core @ 12x 3.7GHz
     /ossssssss/        +ssssooo/-     GPU: NVIDIA GeForce GTX 1050
   `/ossssso+/:-        -:/+osssso+-   RAM: 2875MiB / 32024MiB
  `+sso+:-`                 `.-/+oso: 
 `++:.                           `-/+/
 .`                                 `/
```

**Comment by ZoopOTheGoop** [comment]
This might have coincidentally been fixed by #86 ?

![image](https://user-images.githubusercontent.com/93421308/205211603-0e4af49b-e02d-49b2-8cff-957ba6b9afe2.png)

**Comment by ZoopOTheGoop** [comment]
This works now
